src/common/components/LocalizationProvider.jsx

```diff
@@ -144,6 +144,19 @@ const getDefaultLanguage = () => {
   return 'en';
 };
 
+const getDefaultDateLocale = () => {
```

Comment:
It doesn't make sense to me that we're completely ignoring selected language.

Reply:
This sucks actually. I just tested it on chrome and it behaves differently from firefox. Firefox uses the host os language to render input elements but chrome uses the browser language setting. I guess I will just choose the a language with the expected week format